requirements.txt

```diff
@@ -217,6 +217,7 @@ SQLAlchemy-Continuum==1.2.4
 SQLAlchemy-Utils==0.32.11
 statsd==3.2.1
 traitlets==4.3.1
+ua-parser==0.7.3
```

Comment:
What is the convention about adding dependencies in `setup.py`? As I am unable to find some dependencies from `requirements.txt` in setup.py.

Thanks